nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-queue-listing.js

```diff
@@ -675,6 +692,15 @@
                         nfShell.showPage('provenance?' + $.param({
                                 flowFileUuid: item.uuid
                             }));
+                    } else if (target.hasClass('disabled')) {
+                        nfDialog.showOkDialog({
+                            headerText: 'Empty FlowFile',
+                            dialogContent: 'This FlowFile contains no content to complete this action.'
```

Comment:
Since you are applying the disabled class to this button in this scenario already, what is the motivation of having the popup when the user clicks it? To me this feels unintuitive, I don't think most users would try to click a disabled button, especially when they can see other ones that aren't disabled.

My personal preference would be to either just have nothing, keep button disabled and show no dialog on click, or add a mouse hover bubble saying the same message you have in the popup.  What do you think?

Reply:
@patricker - Original motivation was to support users that may not notice the contrast difference on disabled class, especially when its a single FlowFile in the list. But in the end, I can agree with your point. Will revise to your recommendation and get a commit out shortly.

Reply:
@patricker  - Went ahead and revised per your recommendations and pushed out to the branch. Please take a test drive and let me know if that was what you were looking for. My thanks again for taking a look at this.

Reply:
Thanks. I'm running a build and will test it out.